.github/workflows/bot-pr-updated.yml

```diff
@@ -1,8 +1,10 @@
 on:
   pull_request_target:
     types: [opened,synchronize]
-
 name: OnSyncPR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
```

Comment:
if the head_ref is the ID of last commit, any update or rebase will not use the same head_ref such that they never face the cancel situtation

Reply:
oh right, I forgot to update this, that should be `head_ref`